ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -59,7 +59,7 @@ def test001_cancel_button_edit_no(self, save):
             
     # will continue with us
     @parameterized.expand(['save_btn', 'cancel'])
-    def test002_cancel_button_edit_contact(self, save):
+    def test002_update_contact_with_save_cancel_btn(self, save):
```

Comment:
why you update This test case in this PR, while travis file contains only LIMS-3406

Reply:
Done